packages/client/src/editor/math.ts

```diff
@@ -1,6 +1,12 @@
-import { Coordinates } from '../types';
+import { Coordinates, CoordNumber } from '../types';
+import { coordToNumber, numberToCoord } from '../utils';
 
-export function linePoints(a: Coordinates, b: Coordinates) {
+export function calculateLinePoints(
```

Comment:
NIT: any specific reason for a `function` instead of an arrow-function?

Reply:
`legacy reasons` ^^
Nope, will change.